keyboards/viktus/minne/rules.mk

```diff
@@ -0,0 +1,11 @@
+# Build Options
+SLEEP_LED_ENABLE = no
+BACKLIGHT_ENABLE = no
+RGBLIGHT_ENABLE = yes
+BLUETOOTH_ENABLE = no
+AUDIO_ENABLE = no
+LEADER_ENABLE = no
+MIDI_ENABLE = no
+UNICODE_ENABLE = no
+OLED_DRIVER_ENABLE = no
+LTO_ENABLE = yes
```

Comment:
```suggestion
# This file intentionally left blank
```

Reply:
Is it no longer required to set parts of the firmware you don't want enabled to no in order to save space? Also it has to have "RGBLIGHT_ENABLE = yes" still otherwise it will error when the firmware is compiled. Unless that has changed in just the last few days.

Reply:
Oh sorry, didn't see that. I can also be added to `info.json` under the `features` key as `"rgblight": true`.

Reply:
Oh, I guess I missed that in the documentation. Thank you for pointing that out.